faqs/Tech/what-is-musd.mdx

```diff
@@ -0,0 +1,38 @@
+---
+id: what-is-musd
+sidebar_position: 2
+title: What is MUSD?
+---
+
+import CustomReactPlayer from "../../src/components/CustomReactPlayer";
+import RoundingError from "@site/static/img/faqs/roundingerror.png";
+
+MUSD is a stablecoin that allows users to borrow from the Moria Treasury, a contract on the Bitcoin Cash network. Users must provide, at a minimum 150%, collateral in BCH. MUSD is built on the Moria Protocol.
```

Comment:
-* Clarify that MUSD is made/support by Riften Labs who also did Cauldron (link to that as well) and that it's similar to Liquity on ETH.

Reply:
Fixed.